TemplateApplication/Schedule/ModalView.swift

```diff
@@ -31,17 +31,17 @@ struct ModalView: View {
                 Text(buttonText)
                     .frame(maxWidth: .infinity, minHeight: 38)
             }
-                .padding()
-                .buttonStyle(.borderedProminent)
+            .padding()
+            .buttonStyle(.borderedProminent)
         }
     }
 }
 
 
-struct ModalView_Previews: PreviewProvider {
-    static var previews: some View {
-        ModalView(text: "Preview Modal", buttonText: "Close") {
-            print("Preview Modal closed.")
-        }
+#if DEBUG
+#Preview {
+    ModalView(text: "Preview Modal", buttonText: "Close") {
+        print("Preview Modal closed.")
     }
 }
+#endif
```

Comment:
Might be useful to package this in a sheet as this is probably the main use case and good to test in a preview to see if it works.